scss/details.scss

```diff
@@ -4,19 +4,38 @@
   font-family: 'Roboto';
 }
 
+.badge {
+  height: 120px;
+  left: 225px;
+  position: absolute;
+  transform: rotate(15deg);
+  width: 95px;
+}
+
 .book-container {
   background-color: $white;
   box-shadow: 4px 4px 10px 0 rgba(0, 0, 0, 0.5);
   display: flex;
-  height: 416px;
+  height: 415px;
   width: 900px;
   max-width: 100%;
+
+  &::before {
+    background: url('assets/badge.png');
+    background-size: 100% 100%;
+    content: '';
+    height: 120px;
+    left: 225px;
+    position: absolute;
+    transform: rotate(15deg);
+    width: 95px;
```

Comment:
another way is to use 
`transform: scale(0.2);
  transform-origin: top right;`